src/main/java/io/rocketpartners/cloud/action/sql/SqlDb.java

```diff
@@ -346,6 +349,25 @@ public void delete(Table table, String entityKey) throws Exception
       delete(table, Arrays.asList(entityKey));
    }
 
+   public static Connection getMySqlConn() throws SQLException
```

Comment:
The SqlDb class has a getConnection() method, you don't need to make one that is mysql specific.

Reply:
Removed getMySqlConn() method and used getConnection() method instead